themes/SuiteP/include/EditView/actions_buttons.tpl

```diff
@@ -7,7 +7,7 @@
         {{sugar_button module="$module" id="SAVE" view="$view" form_id="$form_id"}}
         {{sugar_button module="$module" id="CANCEL" view="$view" form_id="$form_id" }}
     {{/if}}
-    {if $showVCRControl}
+    {if $showVCRControl && $view != 'QuickCreate'}
```

Comment:
(original comment on wrong side)
We shouldn't be fixing template issues with very specific use cases (don't want to use the term 'hack' here).  This should be resolved at a higher level i.e. before the template is parsed.